src/vireshJJun20/ConvertCase.java

```diff
@@ -0,0 +1,67 @@
+/*
+ Assignement 29 : 17th July 2020
+
+WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};
+
+*/
+
+package vireshJJun20;
+
+public class ConvertCase {
+
+	static String tempString2 = "";
+
+	// Method accepts Array, converts each index to appropriate case and then saves
+	// in output array.
+	String[] StringConvertCase(String[] inputArray) {
+		String[] outputArray = inputArray;
+		for (int arrIndex = 0; arrIndex < inputArray.length; arrIndex++) {
+			String str = inputArray[arrIndex];
+			String tempString = "";
+			for (int strIndex = 0; strIndex < str.length(); strIndex++) {
+
+				if (str.charAt(strIndex) >= 65 && str.charAt(strIndex) <= 90) {
+					tempString += (char) ((int) str.charAt(strIndex) + 32);
+				} else if (str.charAt(strIndex) >= 'a' && str.charAt(strIndex) <= 'z') {
+					tempString += (char) ((int) str.charAt(strIndex) - 32);
+				} else {
+					tempString2 += str.charAt(strIndex) + " ";
```

Comment:
why are we using the second else. could you please explain?

Reply:
This is used to capture special character Or a digit in an array element.